ext/enchant/enchant.c

```diff
@@ -346,12 +346,12 @@ PHP_FUNCTION(enchant_broker_set_dict_path)
 
 	switch (dict_type) {
 		case PHP_ENCHANT_MYSPELL:
-			enchant_broker_set_param(pbroker->pbroker, "enchant.myspell.dictionary.path", (const char *)value);
+			enchant_broker_set_param(pbroker->pbroker, "enchant.myspell.dictionary.path", value);
```

Comment:
all good, just this call expects a const char * but the compiler should not complain in this case.